File: level76/homework/codwars2.py
#https://www.codewars.com/kata/528d36d7cc451cd7e4000339/train/python (Class გაიხსენეთ და გააკეთეთ მხოლოდ პითონით, ჯერ გაგვივლია js-ში, გადახედეთ Level 60-ს)
def most_money(students):
    total = []
    for student in students:
        total.append((student.fives * 5) + (student.tens * 10) + (student.twenties * 20))
    
    if min(total) == max(total) and len(students) > 1:
        return "all"
    else:
        return students[total.index(max(total))].name

File: level76/homework/codwars2.js
//https://www.codewars.com/kata/528d36d7cc451cd7e4000339/train/javascript
function mostMoney(s) {
  s.sort((x,y)=>sum(y)-sum(x));
  if(s.length>1 && sum(s[0])==sum(s[1]))return 'all';
  return s[0].name;
}

const sum = (s) => (s.fives*5)+(s.tens*10)+(s.twenties*20);